src/GraphQL/TypeExtensions.cs

```diff
@@ -40,6 +40,9 @@ public static bool IsConcrete(this Type type)
         public static bool IsNullable(this Type type)
             => type == typeof(string) || type.IsGenericType && type.GetGenericTypeDefinition() == typeof(Nullable<>);
 
+        /// <summary>
+        /// Determines whether this type is a .NET primitive type (int, bool, etc), or a string, DateTime or DateTimeOffset.
+        /// </summary>
         public static bool IsPrimitive(this Type type)
         {
             return type.IsPrimitive
```

Comment:
Why DateTime?  Why not Guid or Uri?  What about other types?  Do we really need this method at all?

Reply:
Legacy code. Let's remove it.

Reply:
#2327

Reply:
ok